ansible/roles/cleanup_vm/tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Install NFS
```

Comment:
I don't think the clean-up role is the right place for installing packages. It is not obvious behavior.

Instead, I propose either create a new role `nfs-client` and add it to the `vagrant-box.yml` playbook or add a `tasks` block to the playbook and install the package there. Personally, I like the first variant because it helps to keep everything homogeneous.

Reply:
The first variant is much cleaner.